src/Cake.FileHelpers/Cake.FileHelpers.csproj

```diff
@@ -19,7 +19,7 @@
     <Copyright>Copyright 2017-$([System.DateTime]::Now.Year) - Cake Contributions</Copyright>
     <PackageProjectUrl>https://github.com/cake-contrib/Cake.FileHelpers</PackageProjectUrl>
     <PackageIcon>icon.png</PackageIcon>
-    <PackageIconUrl>https://cdn.jsdelivr.net/gh/cake-contrib/graphics/png/cake-contrib-medium.png</PackageIconUrl>
+    <PackageIconUrl>https://cdn.jsdelivr.net/gh/cake-contrib/graphics/png/addin/cake-contrib-addin-medium.png</PackageIconUrl>
```

Comment:
You could remove this line. `CakeContrib.Guidelines` adds this automatically, now.